Marlin/Configuration_adv.h

```diff
@@ -831,40 +831,14 @@
 // Moves (or segments) with fewer steps than this will be joined with the next move
 #define MIN_STEPS_PER_SEGMENT 6
 
-/**
- * Minimum delay after setting the stepper DIR (in ns)
- *    0 : No delay (Expect at least 10µS since one Stepper ISR must transpire)
- *   20 : Minimum for TMC2xxx drivers
- *  200 : Minimum for A4988 drivers
- *  500 : Minimum for LV8729 drivers (guess, no info in datasheet)
- *  650 : Minimum for DRV8825 drivers
- * 1500 : Minimum for TB6600 drivers (guess, no info in datasheet)
- *15000 : Minimum for TB6560 drivers (guess, no info in datasheet)
- */
-#define MINIMUM_STEPPER_DIR_DELAY 0
+// Minimum delay after setting the stepper DIR (in ns)
+//#define MINIMUM_STEPPER_DIR_DELAY
 
-/**
- * Minimum stepper driver pulse width (in µs)
- *   0 : Smallest possible width the MCU can produce, compatible with TMC2xxx drivers
- *   1 : Minimum for A4988 stepper drivers
- *   1 : Minimum for LV8729 stepper drivers
- *   2 : Minimum for DRV8825 stepper drivers
- *   3 : Minimum for TB6600 stepper drivers
- *  30 : Minimum for TB6560 stepper drivers
- */
-#define MINIMUM_STEPPER_PULSE 2
+// Minimum stepper driver pulse width (in µs)
+//#define MINIMUM_STEPPER_PULSE
```

Comment:
I think we should set these to the values for DRV8825, but leave them commented-out, since we mention that DRV8825 is the default on the main driver type config option. And it makes it clear that these have to have a numeric value.